### PR TITLE
Move Webui::WebuiController rescue_from blocks to a concern

### PR DESCRIPTION
**Pull request by vpereira**
Similar with PR #9995 but for Webui Exceptions

### CODE REVIEW

File: src/api/app/controllers/concerns/webui/rescue_handler.rb [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+      if request.xhr?
+        render json: { error: message }, status: 400
+      else
+        flash[:error] = message
+        redirect_back(fallback_location: root_path)
+      end
+    end
+
+    # FIXME: This is more than stupid. Why do we tell the user that something isn't found
```
Comment (dmarcoux) [inline src/api/app/controllers/concerns/webui/rescue_handler.rb:47]: What a lovely comment!

Comment (vpereira) [reply src/api/app/controllers/concerns/webui/rescue_handler.rb:47]: not from me :rofl:

Comment (vpereira) [reply src/api/app/controllers/concerns/webui/rescue_handler.rb:47]: i will drop it

Comment (dmarcoux) [reply src/api/app/controllers/concerns/webui/rescue_handler.rb:47]: > not from me rofl

Yes I know! The point is valid, the tone is quite bad.

Comment (vpereira) [reply src/api/app/controllers/concerns/webui/rescue_handler.rb]: fixed it.

**Pull request merged**